movie_library/utils/serializers/movie_serializer.py

```diff
@@ -12,11 +13,17 @@ class Meta:
 class MovieTrailerSerializer(MovieSerializer):
     trailers = serializers.SerializerMethodField("get_funct", source=Movie)
     trailers = TrailerSerializer(trailers, many=True)
+    actors = serializers.SerializerMethodField("get_function")
+    actors = MovieActorSerializer(actors, many=True)
 
     class Meta:
         model = Movie
-        fields = ['id', 'title', 'duration', 'year', 'release_date', 'created', 'updated', "trailers"]
+        fields = ['id', 'title', 'duration', 'year', 'release_date', 'created', 'updated', "trailers", 'actors']
 
     def get_funct(self, obj):
         name = obj.trailers
         return name
+
+    def get_function(self, obj):
```

Comment:
you have 2 functions named as     def get_function(self, obj):
which one should we call when evaluating the trailers and which one for actor 

USE Meaningful and not duplicate functions names